tgui/packages/tgui/interfaces/Orbit/helpers.ts

```diff
@@ -31,17 +31,7 @@ export function getDisplayName(full_name: string, nickname?: string): string {
   if (!nickname) {
     return full_name;
   }
-
-  if (
-    !full_name?.includes('[') ||
-    full_name.match(/\(as /) ||
-    full_name.match(/^Unknown/)
-  ) {
-    return nickname;
-  }
-
-  // return only the name before the first ' [' or ' ('
-  return `"${full_name.split(/ \[| \(/)[0]}"`;
+  return nickname;
```

Comment:
you gutted this function to a point where it doesnt even need to exist, then didn't delete it despite it no longer being used